dashboard/app/views/levels/show.html.haml

```diff
@@ -22,6 +22,9 @@
   :javascript
     var appOptions = #{app_options.to_json};
   = render partial: 'levels/unplug', locals: {app: @game.app}
+- elsif @level.is_a?(Applab) && app_options['readonlyWorkspace'] && !app_options[:channel]
```

Comment:
is Applab the only type of level that has this problem or do other channel backed levels (like the algebra project levels) need this too?